hawc/apps/summary/views.py

```diff
@@ -163,7 +163,11 @@ class SummaryTableCreate(BaseCreate):
 
     def get_form_kwargs(self):
         kwargs = super().get_form_kwargs()
-        kwargs["table_type"] = self.kwargs["table_type"]
+        try:
+            table_type = constants.TableType(self.kwargs["table_type"])
```

Comment:
Use `.get` or catch `KeyError`? Not sure if this is a potential bug in the security scans.

Reply:
We know that `table_type` exists b/c it's in the url pattern, so the only way it would match this route is if it's valid. The issue we observed were cases where an id of 999 or something like that was passed.